### ISSUE TITLE
Leave incubator context and mature to top-level github-Orga

### ISSUE THREAD

**Issue by MarkusTiede** [baloise/gitopscli#52]
As this is actively used in production I'd like to propose to leave the incubator context and mature to our top-level github-Orga - what do you think?

**Comment by MarkusTiede** [comment]
1:1 I'd like to propose this for the other namespaces being used e.g. on docker-hub.

**Comment by christiansiegel** [comment]
:+1: sounds good to me

**Comment by joachimprinzbach** [comment]
please wait unti the maturity is a bit higher, in terms of docs, testing etc.

**Comment by MarkusTiede** [comment]
@joachimprinzbach would you mind filing issues for leaving the incubator status? Is there something we'd (https://github.com/orgs/baloise/projects/3) do / help to achieve this more quickly?

**Comment by christiansiegel** [comment]
I'd suggest solving the following three main issues before leaving the incubator status:

1. improve documentation: #58, #61 
2. decide on license: #54
3. consolidate enhancement issues (#37, #38, #39, #40 are already partly solved)

**Comment by joachimprinzbach** [comment]
ImO baloise org is not the right organisation for this. There is kind of an eco system with several repos, examples etc. existing in incubator. 

My recommendation would be: Find another home for this.

**Comment by christiansiegel** [comment]
I don't see why this should be an issue. The CLI is self-contained and (soon) well documented. If the baloise org is our destination for mature projects, it is the right place.

We can still refer to the incubator to showcase the CLI in "action".

**Comment by joachimprinzbach** [comment]
Fine for me, was just my opinion.

**Comment by christiansiegel** [comment]
Repo transfer was quite straight forward so far. I only had to replace all hard-coded incubator references in the docs. Travis moved automatically.

I added a placeholder repo to baloise-incubator which redirects to the new Github Page:
https://github.com/baloise-incubator/gitopscli
https://baloise-incubator.github.io/gitopscli/

Also, I opened a PR to switch to the official baloise dockerhub orga: #79
